documentation/samplefiles-dp/samplefiles-dp-for-multi-client/README.md

```diff
@@ -80,6 +80,5 @@ This is not used by Autorest itself.
 swagger-to-sdk:
   - repo: azure-sdk-for-js
   - repo: azure-sdk-for-python
-  - repo: azure-sdk-for-java
```

Comment:
Don't do this. We only need it removed under `specification` directory. Refer to https://github.com/Azure/azure-rest-api-specs/pull/33888

Reply:
Fixed by the [commit#bca2bfc](https://github.com/Azure/azure-rest-api-specs/pull/34006/commits/bca2bfc9e87566504f19abe24884eede9096498d)